src/lightning_utilities/__about__.py

```diff
@@ -1,6 +1,6 @@
 import time
 
-__version__ = "0.2.0"
+__version__ = "0.3.0"
```

Comment:
```suggestion
__version__ = "0.3.0dev"
```

Reply:
I want to do a new release today or tomorrow, so this will need to change to 0.3.0 anyways